lib/openrtb2_3/site.js

```diff
@@ -0,0 +1,123 @@
+var RtbObject = require('../rtbObject');
+var PublisherBuilder = require('./publisher').builder;
+var removeEmptyValues = require('../utils').removeEmptyValues;
+
+var Site = function(id, name, domain, cat, sectionCat, pageCat, page, ref, search, mobile, privacypolicy, publisher, keyworkds, ext){
+  this.id = id;
+  this.name = name;
+  this.domain = domain;
+  this.cat = cat;
+  this.sectionCat = sectionCat;
+  this.pageCat = pageCat;
+  this.page = page;
+  this.ref = ref;
+  this.search = search;
+  this.mobile = mobile;
+  this.privacypolicy = privacypolicy;
+  this.publisher = publisher;
+  this.keyworkds = keyworkds;
+  this.ext = ext;
+};
+
+Site.prototype = Object.create(RtbObject.prototype);
+
+var SiteBuilder = function(){};
+
+SiteBuilder.prototype.id = function(id){
+  this._id = id;
+  return this;
+};
+
+SiteBuilder.prototype.name = function(name){
+  this._name = name;
+  return this;
+};
+
+SiteBuilder.prototype.domain = function(domain){
+  this._domain = domain;
+  return this;
+};
+
+SiteBuilder.prototype.cat = function(cat){
+  this._cat = cat;
+  return this;
+};
+
+SiteBuilder.prototype.sectionCat = function(sectionCat){
```

Comment:
The convention we generally used was to follow the namong convention of OpenRTB. So I think it's better if we call `sectionCat` --> `sectioncat`.

Believe me I prefer camelcase too but OpenRTB decided otherwise :)